.github/workflows/build.yml

```diff
@@ -91,6 +91,12 @@ jobs:
         run: |
           echo "yearmonth=$(/bin/date -u '+%Y-%m')" >> "$GITHUB_OUTPUT"
         shell: bash
+      - name: Sanitize the branch name
+        # So that if we end up using the branch name in the artifact name we won't have any unexpected chars in the name
```

Comment:
Thanks for the change, it makes sense. But I don't think this comment it's that helpful.
Could you mention instead that we use it for distinguishing reports for different branches, and maybe add an example of the conversion.  For example, `wip/4.2` becomes `wip-4.2`.